LICENSE.md

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
```

Comment:
I think we use BSD.

We should make sure this is standardized, but at the least, be careful not to add new licenses